src/_pytest/stepwise.py

```diff
@@ -72,7 +72,7 @@ def pytest_collection_modifyitems(self, session, config, items):
 
     def pytest_runtest_logreport(self, report):
         # Skip this hook if plugin is not active or the test is xfailed.
```

Comment:
This comment seems wrong

Reply:
oops indeed

Reply:
https://github.com/pytest-dev/pytest/pull/5560